### PR TITLE
Remove extra "[]" from OpenStack templates

### PR DESCRIPTION
**Pull request by nurfet-becirevic**
This is a WIP associated with the issue [1071](https://github.com/coreos/tectonic-installer/issues/1071) for OpenStack provider.

@coreypobrien, @lblackstone, @jamiehannaford : would you guys be able to test the changes?

### CODE REVIEW

File: platforms/openstack/neutron/main.tf [hunk 1]
```diff
@@ -96,7 +96,7 @@ EOF
   base_domain           = "${var.tectonic_base_domain}"
   cluster_name          = "${var.tectonic_cluster_name}"
   container_image       = "${var.tectonic_container_images["etcd"]}"
-  core_public_keys      = ["${module.secrets.core_public_key_openssh}"]
```
Comment (lblackstone) [inline platforms/openstack/neutron/main.tf]: Appears that the public key changes aren't working:
```
* module.master_nodes.var.core_public_keys: variable core_public_keys in module master_nodes should be type list, got string
* module.etcd.var.core_public_keys: variable core_public_keys in module etcd should be type list, got string
* module.worker_nodes.var.core_public_keys: variable core_public_keys in module worker_nodes should be type list, got string```

**Comment by coreypobrien** [comment]
@nurfet-becirevic looks like the `core_public_keys` need to keep them:
```
* module.master_nodes.var.core_public_keys: variable core_public_keys in module master_nodes should be type list, got string
* module.etcd.var.core_public_keys: variable core_public_keys in module etcd should be type list, got string
* module.worker_nodes.var.core_public_keys: variable core_public_keys in module worker_nodes should be type list, got string
```

**Comment by nurfet-becirevic** [comment]
ok to test

**Comment by jasmingacic** [comment]
ok to test

**Comment by lblackstone** [comment]
@coreos/team-installer OpenStack only, good to merge

Tested with a fast-forward merge on master.

**Comment by squat** [comment]
thanks @nurfet-becirevic and @lblackstone!

**Pull request merged**